src/pages/troubleshooting/logging-monitoring.md

```diff
@@ -50,7 +51,7 @@ ACD runtime exceptions | `kubernetes.container_name:"ibm-wh-acd-*" AND exception
   a. Red Hat OpenShift logging operator
   b. OpenShift Elasticsearch operator
   
-Logs including JSON logs are usually represented as a string inside the message field. That makes it hard for users to query specific fields inside a JSON document. OpenShift Logging’s Log Forwarding API enables you to parse JSON logs into a structured object and forward them to either OpenShift Logging-managed Elasticsearch or any other third-party system supported by the Log Forwarding API
+Logs including JSON logs are usually represented as a string inside the message field. That makes it hard for users to query specific fields inside a JSON document. OpenShift Logging\u2019s Log Forwarding API enables you to parse JSON logs into a structured object and forward them to either OpenShift Logging-managed Elasticsearch or any other third-party system supported by the Log Forwarding API
```

Comment:
Looks good, Tom. I'm not sure about the \u2019 character above. I don't think the single apostrophe was a problem before so maybe that's not necessary?

Reply:
Good catch.  I think it will still work but I put it back.  It must have gotten translated when I copied it to visual studio code to  for editing.